services/movie_session.py

```diff
@@ -42,3 +42,12 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict]:
+    movie_session = MovieSession.objects.get(id=movie_session_id)
+    tickets = movie_session.tickets.all()
+    taken_seats = [
+        {"row": ticket.row, "seat": ticket.seat} for ticket in tickets
+    ]
```

Comment:
You could implement this method in one line using Django ORM and `values` or `values_list`. It allows to avoid using list comprehension.